roles/tests/client.js

```diff
@@ -28,19 +28,22 @@ describe('roles', function () {
     }
   }
 
-  function testUser (username, expectedRoles, group) {
+  function testUser (username, expectedRoles, scope) {
     var user = users[username]
-
+    // console.log(`testUser client scope=${typeof scope}`);
+    if(typeof scope == 'string') {
```

Comment:
Is there a reason for changing this ..? It should work either way, shouldn't it? Now it seems like we test one case less ...